### ISSUE TITLE
pdf import loses whitespace in some cases

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-transform#314]
**Describe the bug**
Transforming pdf to commonmark loses whitespace in some instances.

**To Reproduce**
1. Go to `./markdown-transform/packages/markdown-pdf
2. Test with the Land sale contract:
     ```
     ../markdown-cli/index.js transform --from pdf --to markdown --input test/data/Land_Sale_Contract.pdf 
     ```
3. The input is:
<img width="914" alt="Screenshot 2020-08-31 at 2 20 24 PM" src="https://user-images.githubusercontent.com/670099/91753046-4942f200-eb95-11ea-9cc4-584c3c374353.png">
4. But the output shows:
```
../markdown-cli/index.js transform --from pdf --to markdown --input test/data/Land_Sale_Contract.pdf 
2:18:26 PM - info: 
Page __ of __


www.PrintableContracts.com


**LAND SALE CONTRACT**


The first party(henceforth knownas “Buyer”) agrees to purchasefrom the 
second party(henceforth known as “Seller”) the land(henceforth known as 
“Land”) located at the following address: **{address}**,in the city of **{city}**, countyof 
**{county}**, and State of**{state}**. Further legal description of the Landis as follows:
```

**Comment by DianaLease** [comment]
This is the issue from the `pdf2json` library we are using that @dselman mentioned: https://github.com/modesty/pdf2json/issues/94

**Comment by dselman** [comment]
Please retest once https://github.com/accordproject/markdown-transform/pull/315 is merged.

**Comment by jeromesimeon** [comment]
Addressed (to some extent) in #315
